server/controllers/system/edit-department.php

```diff
@@ -53,6 +53,10 @@ public function handler() {
 
         $departmentInstance = Department::getDataStore($departmentId);
 
+        if($departmentId == Setting::getSetting('default-department-id')->getValue()){
```

Comment:
Aren't you missing `$private` in this condition? You're now rejecting any default department edition